litert/vendors/qualcomm/core/builders/conv2d_op_builder.cc

```diff
@@ -93,6 +95,18 @@ std::vector<OpWrapper> BuildConv2dOp(
     transpose_op.AddTensorParam(QNN_OP_TRANSPOSE_PARAM_PERM, permute_tensor);
   }
 
+  TensorWrapper* conv2d_output_tensor = nullptr;
+
+  // If Conv2d fused activation is available, create a new tensor for the
+  // output of Conv2d, which is also the input of the activation op.
+  if (fused_activation != kLiteRtFusedActivationNone) {
+    conv2d_output_tensor = &tensor_pool.CreateNativeTensor(
+        outputs[kOutputIndex].get().GetDataType(), {},
```

Comment:
Also need to keep quantization param.
```suggestion
        outputs[kOutputIndex].get().GetDataType(), outputs[kOutputIndex].get().GetQuantParams(),
```